### ISSUE TITLE
inconsistent range notation

### ISSUE THREAD

**Issue by Seasawher** [lecopivo/SciLean#37]
`Std.range` (for example `[1:5]`) does not include right edge.

but notation `[-k:k]` in `Float^[[-k:k]]` does include right edge, because which is defined as ICC.

I thought this is misleading notation.

**Comment by lecopivo** [comment]
Yeah I know, any idea how to improve the notation? Would you prefer  `Float^[[-k:k+1]]`? In math, you would write `[-k,k]` for `Icc` or `[-k,k)` for `Ico`. 

Maybe not include this notation at all and you have to write `Float^[Icc -k k]`

**Comment by Seasawher** [comment]
how about `[-k ≤ · ≤ k]`?
